src/System.Threading.AccessControl/tests/ThreadingAclExtensionsTests.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.AccessControl;
+using Xunit;
+
+namespace System.Threading.Tests
+{
+    public static class ThreadingAclExtensionsTests
+    {
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public static void ExistenceTest_Windows()
+        {
+            var e = new ManualResetEvent(true);
+            var s = new Semaphore(1, 1);
+            var m = new Mutex();
+
+            e.GetAccessControl();
+            e.SetAccessControl(new EventWaitHandleSecurity());
+            s.GetAccessControl();
+            s.SetAccessControl(new SemaphoreSecurity());
+            m.GetAccessControl();
+            m.SetAccessControl(new MutexSecurity());
+        }
```

Comment:
Is there anything we can assert about the result of these calls?  At the very least seems like we could Assert.NotNull?